s3torchconnectorclient/python/tst/integration/test_mountpoint_s3_integration.py

```diff
@@ -15,369 +15,244 @@
 
-test_config = get_test_config()
-
 HELLO_WORLD_DATA = b"Hello, World!\n"
 TEST_USER_AGENT_PREFIX = "integration-tests"
 
 
-@pytest.mark.parametrize(
-    "region, bucket, key",
-    [
-        (test_config.region, test_config.bucket, "sample-files/hello_world.txt"),
-        (
-            test_config.express_region,
-            test_config.express_bucket,
-            "sample-files/hello_world.txt",
-        ),
-    ],
-)
-def test_get_object(region: str, bucket: str, key: str):
-    client = MountpointS3Client(region, TEST_USER_AGENT_PREFIX)
-    stream = client.get_object(bucket, key)
+def test_get_object(sample_directory):
+    client = MountpointS3Client(sample_directory.region, TEST_USER_AGENT_PREFIX)
+    stream = client.get_object(
+        sample_directory.bucket, f"{sample_directory.prefix}hello_world.txt"
+    )
 
     full_data = b"".join(stream)
     assert full_data == HELLO_WORLD_DATA
 
 
-@pytest.mark.parametrize(
-    "region, bucket, key",
-    [
-        (test_config.region, test_config.bucket, "sample-files/hello_world.txt"),
-        (
-            test_config.express_region,
-            test_config.express_bucket,
-            "sample-files/hello_world.txt",
-        ),
-    ],
-)
-def test_get_object_with_unpickled_client(region: str, bucket: str, key: str):
-    original_client = MountpointS3Client(region, TEST_USER_AGENT_PREFIX)
+def test_get_object_with_unpickled_client(sample_directory):
+    original_client = MountpointS3Client(
+        sample_directory.region, TEST_USER_AGENT_PREFIX
+    )
     assert original_client.user_agent_prefix == TEST_USER_AGENT_PREFIX
     pickled_client = pickle.dumps(original_client)
     assert isinstance(pickled_client, bytes)
     unpickled_client = pickle.loads(pickled_client)
     assert unpickled_client.user_agent_prefix == TEST_USER_AGENT_PREFIX
     stream = unpickled_client.get_object(
-        bucket,
-        key,
+        sample_directory.bucket, f"{sample_directory.prefix}hello_world.txt"
     )
     full_data = b"".join(stream)
     assert full_data == HELLO_WORLD_DATA
 
 
-@pytest.mark.parametrize(
-    "region, bucket, key",
-    [
-        (
-            test_config.region,
-            f"{test_config.bucket}-invalid",
-            "sample-files/hello_world.txt",
-        ),
-        (
-            test_config.express_region,
-            f"{test_config.express_bucket}-invalid",
-            "sample-files/hello_world.txt",
-        ),
-    ],
-)
-def test_get_object_invalid_bucket(region: str, bucket: str, key: str):
-    client = MountpointS3Client(region, TEST_USER_AGENT_PREFIX)
+def test_get_object_invalid_bucket(sample_directory):
+    client = MountpointS3Client(sample_directory.region, TEST_USER_AGENT_PREFIX)
     with pytest.raises(S3Exception) as error:
-        next(client.get_object(f"{bucket}-{uuid.uuid4()}", key))
+        next(
+            client.get_object(
+                f"{sample_directory.bucket}-{uuid.uuid4()}", sample_directory.prefix
```

Comment:
I think this is missing the key, but that shouldn't affect the outcome of the test